proc-macros/src/helpers.rs

```diff
@@ -87,7 +87,7 @@ fn find_jsonrpsee_crate(crate_names: &[&str]) -> Result<proc_macro2::TokenStream
 ///  use jsonrpsee::core::{RpcResult, SubscriptionResult};
 ///
 ///  #[rpc(client, server)]
-///  pub trait RpcTrait<A, B, C> {
+///  pub trait RpcTrait<A, B: Clone, C> {
```

Comment:
Out of interest, why did this need adding?

Reply:
(some question for some other `Clone`'s. Is it to do with the `ToOwned` bound on the response? Will this make using the macro more painful?)

Reply:
Every response for the server needs to implement Clone/ToOwned

such as

```
fn call() -> Result<T, Erro>;
```

T will be used in `IntoResponse trait` and thus must implement Clone.

Yes, it will be more annoying to use but Alex added these custom bounds on the traits which is okay I guess.

Reply:
it's a trade-off all these annoyance can away with separate serialize/deserialize types :)

That's why I skipped these complicated stuff in the first place but reduces some technical debt with duplicate types and hard to read.